src/main/java/Duke.java

```diff
@@ -8,81 +12,58 @@
     public static List<TaskDTO> taskList = new ArrayList<>();
     public static int taskId = 0;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
 
-        System.out.println("____________________________________________________________");
-        System.out.println("Hello! I'm Duke\n" +
-                "What can I do for you?");
 
         Scanner scanner = new Scanner(System.in);
 
-        processCommand(scanner);
-
-    }
+        UI ui = new UI();
+        UI.printWelcomeMessage();
+        String command = ui.readCommand();
 
-    public static void processCommand (Scanner scanner){
-        String command = scanner.nextLine();
-        TaskDTO newTask = new TaskDTO();
-        System.out.println("____________________________________________________________");
-        if (command.equalsIgnoreCase("list")){
```

Comment:
Nice use of Ignore Case, perhaps I should implement on my Code instead of changing command to Lower Case. Thanks!